common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4418,7 +4419,7 @@ object CelebornConf extends Logging {
       .version("0.3.0")
       .doc("Interval for client to check expired shuffles.")
       .timeConf(TimeUnit.MILLISECONDS)
-      .createWithDefaultString("60s")
+      .createWithDefaultString("20s")
```

Comment:
Why is the default value changed? Any production practice?

Reply:
Draft need to discuss

Reply:
Agree, let us keep the defaults unless there are good reasons to change it.